goalgorilla/open_social
# #2918633 by maikelkoopman: Remove color as background of logo

When uploading a logo with transparency the default blue color is seen behind it.

How to test:
- Upload a custom logo to the site, this must a png with transparency
- You see no "primary" background color in the logo.

https://www.drupal.org/node/2918633
